src/base/bittorrent/session.cpp

```diff
@@ -4717,6 +4719,29 @@ void Session::startUpTorrents()
             }
         }
 
+        const TagSet tag = resumeData.tags;
```

Comment:
It isn't a single tag.

Reply:
Yes I just realized there can be multiple tags.